backend/routers/Pins.py

```
@@ -0,0 +1,127 @@
+ALGORITHM = "HS256"
+oauth2_scheme = OAuth2PasswordBearer(tokenUrl="auth/login")
+
+def get_current_user(token: str = Depends(oauth2_scheme), db: Session = Depends(get_db)):
+    credentials_exception = HTTPException(
+        status_code=status.HTTP_401_UNAUTHORIZED,
+        detail="Could not validate credentials",
+        headers={"WWW-Authenticate": "Bearer"},
+    )
+    try:
+        if not SECRET_KEY:
+             raise RuntimeError("SECRET_KEY must be set in environment")
```

Ruff (0.14.8)

Avoid specifying long messages outside the exception class

(TRY003)